src/components/ApiService/ApiService.js

```diff
@@ -0,0 +1,18 @@
+import axios from 'axios'
+
+const urlport = `${process.env.REACT_APP_BASEURL}:${process.env.REACT_APP_PORT}`
+
+export const getAll = (params,query) => {
+    const URL = `${urlport}${params}${query}`;
+    return axios(URL, {
```

Comment:
`axios.get(url', {
    params: {
      ID: 12345
    }
  })
  .then(function (response) {
    console.log(response);
  })
  .catch(function (error) {
    console.log(error);
  });`

This would be more readable than putting url, params and query all together.